browser-tests/data/pagesAndContexts.ts

```diff
@@ -57,7 +57,8 @@ export const pagesToTest: Record<string, (string | undefined)[]> = {
   "update-details-failed": ["repeatFraudCheck",undefined],
   "update-details": [],
   "update-name-date-birth": ["repeatFraudCheck","reuse","rfcAccountDeletion",undefined],
-  "uk-driving-licence-details-not-correct": []
+  "uk-driving-licence-details-not-correct": [],
+  "prove-identity-another-way": ["coiAuthSourceCheck", undefined],
```

Comment:
I would kinda prefer these context values to avoid talking about initiatives like this, and instead reflect what core-front actually cares about... We might want to use this elsewhere, where `coiAuthSourceCheck` makes no sense! Perhaps something like `noF2f`?

Compare to what we did for the fast follow changes - `rfc` -> `existingIdentityInvalid` etc.

(Speaking of - should those be listed here under `sorry-could-not-confirm-details`?)